Packs/MISP/Integrations/MISPV3/MISPV3_test.py

```diff
@@ -344,6 +344,30 @@ def test_convert_arg_to_misp_args(mocker):
     assert convert_arg_to_misp_args(args, args_names) == [{'dst-port': 8001}, {'src-port': 8002}, {'name': 'test'}]
 
 
+@pytest.mark.parametrize('dbot_type, dbot_score_type, value, expected_type_object', [
```

Comment:
Please add a test that shows the senario search_warninglists is true

Reply:
Sorry for the delayed response. I was ill the last days. I've added a corresponding test case.

Reply:
Hope you feel better now :)
Great work @nikstuckenbrock !